### PR TITLE
golint fix TLs->TLS in docker/

### PR DESCRIPTION
**Pull request by sevki**
golint errors mentioned in #14756

```
docker/flags.go:29:2: var dockerTlsVerify should be dockerTLSVerify
docker/flags.go:94:2: var flTls should be flTLS
docker/flags.go:96:2: var flTlsVerify should be flTLSVerify
```

### CODE REVIEW

File: docker/daemon.go [hunk 1]
```diff
@@ -123,8 +123,8 @@ func mainDaemon() {
 	}
 	serverConfig = setPlatformServerConfig(serverConfig, daemonCfg)
 
-	if *flTls {
-		if *flTlsVerify {
+	if *flTlS {
```
Comment (runcom) [inline docker/daemon.go]: should this be `TLS`? :)

Comment (sevki) [inline docker/daemon.go]: :smile: just noticed that. need to switch to fonts

**Comment by runcom** [comment]
LGTM if janky's happy

**Comment by sevki** [comment]
@runcom :tada: my first PR to docker.

**Comment by sevki** [comment]
@runcom had a [side project](https://sevki.org/joker) that [commented inline on golint issues](https://github.com/sevki/docker/commit/37a6f9bc6b52fa5ce1ff5e30f1b8112ae8c34654). Should I include a script to run that against commits in this PR too?

**Comment by runcom** [comment]
@sevki interesting, we may need to discuss that so either create a PR or just come on IRC to chat (@jfrazelle may have more to say about including your script also)

**Comment by sevki** [comment]
@runcom well I say script but it's basically this 
```
joker -repo=docker -owner=docker -token={token} -commit=`git describe --always` -scanner=golint
```
don't know if it's PR worthy, I'll hop on IRC

**Comment by calavera** [comment]
I think we don't want to run golint just yet because we have too much technical debt that doesn't match the go linter. Either way that should be another PR.

Thanks for fixing this issue. LGTM. Merging!

**Comment by sevki** [comment]
@calavera that app only works on modified or added lines in a commit, doesn't lint existing stuff. so no noise, it's just about preventing new stuff from coming in.

**Comment by calavera** [comment]
@sevki that sounds good, we could integrate it in https://github.com/jfrazelle/leeroy. We might want to post only one comment with all the warnings instead of line by line though.

File: docker/flags.go [hunk 2]
```diff
@@ -26,7 +26,7 @@ func (a byName) Less(i, j int) bool { return a[i].name < a[j].name }
 
 var (
 	dockerCertPath  = os.Getenv("DOCKER_CERT_PATH")
-	dockerTlsVerify = os.Getenv("DOCKER_TLS_VERIFY") != ""
+	dockerTlSVerify = os.Getenv("DOCKER_TLS_VERIFY") != ""
```
Comment (tiborvass) [inline docker/flags.go]: @sevki @runcom @calavera you missed this one :P it should be `TLS` not `TlS` ;)

**Pull request merged**